annotated-projects/PXD000612/sdrf.tsv

```diff
@@ -1,316 +1,316 @@
-sample 29	homo sapiens	not applicable	not available	not available	Female	Nocodazole	compound: drug Nocodazole	not applicable	run 313	AC=MS:1002038;NM=label free sample	AC=MS:1001911;NM=Q Exactive	20 ppm	0.05 Da	AC=MS:1000422;NM=HCD	AC=MS:1001251;NM=Trypsin	NM=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NM=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=Variable	"NM=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:60"	14	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/08/PXD000612/pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH5.raw	pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH5.raw	cell	Nocodazole
-sample 29	homo sapiens	not applicable	not available	not available	Female	Nocodazole	compound: drug Nocodazole	not applicable	run 314	AC=MS:1002038;NM=label free sample	AC=MS:1001911;NM=Q Exactive	20 ppm	0.05 Da	AC=MS:1000422;NM=HCD	AC=MS:1001251;NM=Trypsin	NM=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NM=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=Variable	"NM=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:61"	15	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/08/PXD000612/pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH6.raw	pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH6.raw	cell	Nocodazole
-sample 29	homo sapiens	not applicable	not available	not available	Female	Nocodazole	compound: drug Nocodazole	not applicable	run 315	AC=MS:1002038;NM=label free sample	AC=MS:1001911;NM=Q Exactive	20 ppm	0.05 Da	AC=MS:1000422;NM=HCD	AC=MS:1001251;NM=Trypsin	NM=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NM=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=Variable	"NM=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:62"	16	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/08/PXD000612/pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH8.raw	pY_EXQ5_KiSh_SA_LabelFree_HeLa_Proteome_Nocodazole_rep4_pH8.raw	cell	Nocodazole
\ No newline at end of file
+sample 1	homo sapiens	not applicable	not available	not available	Female	normal	compound: none	not applicable	run 1	AC=MS:1002038;NT=label free sample	AC=MS:1001911;NT=Q Exactive	20 ppm	0.05 Da	AC=MS:1000422;NT=HCD	AC=MS:1001251;NT=Trypsin	NT=Oxidation;MT=Variable;TA=M;AC=UNIMOD:35	NT=Acetyl;AC=UNIMOD:1;PP=Protein N-term;MT=Variable	"NT=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:21"	1	ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/08/PXD000612/20111225_EXQ5_KiSh_SA_LabelFree_HeLa_Phospho_Control_rep4_Fr1.raw	20111225_EXQ5_KiSh_SA_LabelFree_HeLa_Phospho_Control_rep4_Fr1.raw	cell	normal
```

Comment:
"NT=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:21"
should be
NT=Phospho;MT=Variable;TA=S,T,Y;AC=UNIMOD:21